sdk/typescript/runtime/config.go

```diff
@@ -131,14 +132,26 @@ func analyzeModuleConfig(ctx context.Context, modSource *dagger.ModuleSource) (c
 
 	// If a first init, there will be no directory, so we ignore the error here.
 	// We also only include package.json & lockfiles to benefit from caching.
-	cfg.source = modSource.ContextDirectory().Directory(cfg.subPath)
-	configEntries, err := dag.Directory().WithDirectory(".", cfg.source, dagger.DirectoryWithDirectoryOpts{
-		Include: moduleConfigFiles("."),
-	}).Entries(ctx)
-	if err == nil {
-		for _, entry := range configEntries {
-			cfg.entries[entry] = true
-		}
+	matches, err := modSource.ContextDirectory().Glob(ctx, cfg.subPath+"/**")
```

Comment:
nit: 
```suggestion
	matches, err := modSource.ContextDirectory().Directory(cfg.subPath).Entries(ctx)
```

doesn't seem necessary to glob, which is gonna walk way more files+directories.